src/kohlrahbi/__init__.py

```diff
@@ -104,56 +106,93 @@ def load_all_known_pruefis_from_file(
     return pruefis
 
 
-@click.command()
-@click.option(
-    "-p",
-    "--pruefis",
-    default=[],
-    required=False,
-    help="Five digit number like 11042 or use wildcards like 110* or *042 or 11?42.",
-    multiple=True,
-)
-@click.option(
-    "-i",
-    "--input_path",
-    type=click.Path(exists=True, dir_okay=True, file_okay=False, path_type=Path),
-    prompt="Input directory",
-    help="Define the path to the folder with the docx AHBs.",
-)
-@click.option(
-    "-o",
-    "--output_path",
-    type=click.Path(exists=False, dir_okay=True, file_okay=False, path_type=Path),
-    default="output",
-    prompt="Output directory",
-    help="Define the path where you want to save the generated files.",
-)
-@click.option(
-    "--file-type",
-    type=click.Choice(["flatahb", "csv", "xlsx", "conditions"], case_sensitive=False),
-    multiple=True,
-)
-@click.option(
-    "--assume-yes",
-    "-y",
-    is_flag=True,
-    help="Confirm all prompts automatically.",
-)
-# pylint: disable=too-many-branches, too-many-statements, too-many-locals
-def main(pruefis: list[str], input_path: Path, output_path: Path, file_type: list[str], assume_yes: bool):
+def create_sheet_name(filename: str):
     """
-    A program to get a machine readable version of the AHBs docx files published by edi@energy.
+    Creates a sheet name from the filename.
+
+    We need to shorten the sheet name because Excel only allows 31 characters for sheet names.
+    This function replaces some words with acronyms and removes some words.
     """
-    check_python_version()
+    sheet_name = filename.split("-informatorischeLesefassung")[0]
+
+    if "Entscheidungsbaum-Diagramm" in sheet_name:
+        sheet_name = sheet_name.replace("Entscheidungsbaum", "EBDs")
+    if "Artikelnummern" in sheet_name:
+        sheet_name = sheet_name.replace("Artikelnummern", "Artikelnr")
+    if "Codeliste" in sheet_name:
+        sheet_name = sheet_name.replace("Codeliste", "CL")
+    if len(sheet_name) > 31:
+        # Excel only allows 31 characters for sheet names
+        # but REQOTEQUOTESORDERSORDRSPORDCHGAHB is 33 characters long
+        sheet_name = sheet_name.replace("HG", "")
```

Comment:
why? can you add a comment for this replacement?

Reply:
The reason is mentioned two lines above. Should I make it more visible?